generator/move_cross_mwm/move_cross_mwm.cpp

```diff
@@ -0,0 +1,76 @@
+#include "generator/routing_helpers.hpp"
+
+#include "routing/cross_mwm_connector_serialization.hpp"
+
+#include "coding/file_container.hpp"
+
+#include "defines.hpp"
+
+#include "3party/gflags/src/gflags/gflags.h"
+
+DEFINE_string(old_mwm, "", "Path to the old mwm file.");
+DEFINE_string(new_mwm, "", "Path to the new mwm file.");
+DEFINE_string(osm2ft_path, "", "Path to an osm2ft file for the new mwm.");
+
+using namespace routing;
+
+int main(int argc, char ** argv)
+{
+  google::SetUsageMessage(
+      "Move a crossmwm section from one mwm to another, using an osm2ft file to renumber "
+      "features.");
+  google::ParseCommandLineFlags(&argc, &argv, true);
+
+  if (FLAGS_old_mwm.empty() || FLAGS_new_mwm.empty())
+  {
+    google::ShowUsageWithFlagsRestrict("Move Cross MWM", "move_cross_mwm");
```

Comment:
Please, implement flags validators instead - see openlr/openlr_stat/openlr_stat.cpp for reference.